README.md

```diff
@@ -18,6 +18,7 @@ MMPose is an open-source toolbox for pose estimation based on PyTorch.
 It is a part of the [OpenMMLab project](https://github.com/open-mmlab).
 
 The master branch works with **PyTorch 1.3+**.
+Please kindly note that MMPose will switch to **PyTorch 1.5+** from Oct. 2021. The compatibility to earlier versions of PyTorch will no longer be guaranteed.
```

Comment:
Modify L20 to the current value

Reply:
What does the current value mean?

Reply:
![image](https://user-images.githubusercontent.com/9464825/131768340-fa8d0270-cb5b-4d28-bd24-ec9d1a9ae75a.png)

Reply:
Okay the modification could be delayed to October